examples/plot_knockoff_aggregation.py

```diff
@@ -11,22 +11,20 @@
 This example exhibits the two aggregation procedures described
 by :footcite:t:`pmlr-v119-nguyen20a` and :footcite:t:`Ren_2023` to derandomize
 inference.
```

Comment:
I suggest changing 
 > "A notable drawback of this procedure is the randomness associated with the knockoff generation process. This can result in unstable inference."

to 

> "A notable drawback of this procedure is the randomness associated with generating knockoff variables. This can result in fluctuations of the statistical power and false discovery proportion, and consequently, unstable inference.